helpline-dialer.php

```diff
@@ -14,7 +14,7 @@
     if ($call_status != "completed") { ?>
         <Dial method="GET"
           timeout="<?php echo $call_timeout?>"
-          callerId="<?php echo isset($_REQUEST["Called"]) ? $_REQUEST["Called"] : "0000000000" ?>"
+          callerId="<?php if (isset($GLOBALS['outbound_callerid']) && $GLOBALS['outbound_callerid']) { echo isset($GLOBALS["outbound_callerid"]) ? $GLOBALS["outbound_callerid"] : "0000000000"; } else { echo isset($_REQUEST["Called"]) ? $_REQUEST["Called"] : "0000000000";  } ?>"
```

Comment:
This line of code is getting more complicated to read.  Can you move this into a proper if/else block with variable assignment?